integration/tests/cook/test_basic.py

```diff
@@ -871,4 +866,4 @@ def check_unscheduled_reason():
                 self.assertEqual(job_uuid, unscheduled_jobs['uuid'])
             check_unscheduled_reason()
         finally:
-            util.kill_job(self.cook_url, job_uuid)
+            util.cancel_jobs(self.cook_url, [job_uuid])
```

Comment:
Is the change from "kill" to "cancel" intentional?

Reply:
I think I named this method "cancel_jobs" because we have a "test_cancel_job" test that I was looking at when I wrote it. I renamed this method to kill_jobs now that you pointed out the inconsistency with our usual terminology we use for Cook.